### PR TITLE
[MISC] Enable COS Integration tests

### PR DESCRIPTION
**Pull request by deusebio**

### CODE REVIEW

File: releases/3.4/bundle.yaml.j2 [hunk 1]
```diff
@@ -46,16 +46,16 @@ applications:
     constraints: arch=amd64
   history-server:
     charm: spark-history-server-k8s
-    channel: 3.4/stable
-    revision: 15
+    channel: 3.4/edge
+    revision: 24
     resources:
-      spark-history-server-image: 9
+      spark-history-server-image: 12
```
Comment (deusebio) [inline releases/3.4/bundle.yaml.j2:52]: TMP revision from PR: https://github.com/canonical/spark-history-server-k8s-operator/pull/40

File: releases/3.4/bundle.yaml.j2 [hunk 2]
```diff
@@ -46,16 +46,16 @@ applications:
...
     scale: 1
     constraints: arch=amd64
   integration-hub:
     charm: spark-integration-hub-k8s
     channel: latest/edge
-    revision: 3
+    revision: 4
```
Comment (deusebio) [inline releases/3.4/bundle.yaml.j2:58]: published version [here](https://github.com/canonical/spark-integration-hub-k8s-operator/releases/tag/rev4)

File: releases/3.4/bundle.yaml.j2 [hunk 3]
```diff
@@ -2,8 +2,8 @@ bundle: kubernetes
 applications:
   kyuubi:
     charm: kyuubi-k8s
-    channel: "latest/edge"
-    revision: 5
+    channel: latest/edge
+    revision: 6
```
Comment (deusebio) [inline releases/3.4/bundle.yaml.j2:6]: published version [here](https://github.com/canonical/kyuubi-k8s-operator/releases/tag/rev6)

**Review by welpaolo — APPROVED** [review]
LGTM

File: .github/workflows/ci-tests.yaml [hunk 4]
```diff
@@ -42,14 +42,15 @@ jobs:
 
   integration-tests:
     name: Integration Tests
-    runs-on: [ "self-hosted", "linux", "X64", "jammy", "large" ]
+    runs-on: ubuntu-22.04
+    # [ "self-hosted", "linux", "X64", "jammy", "large" ]
```
Comment (welpaolo) [inline .github/workflows/ci-tests.yaml:46]: Do we want to keep this line?

**Review by welpaolo — APPROVED** [review]

File: .github/workflows/ci-tests.yaml (hunk 4 above)
Comment (deusebio) [reply .github/workflows/ci-tests.yaml:46]: naaa, removing this

Comment (deusebio) [reply .github/workflows/ci-tests.yaml:46]: I'll remove this in the final PR, just to avoid re-triggering the CI if not needed

**Review by theoctober19th — APPROVED** [review]

**Pull request merged**